index.src.html

```diff
@@ -394,6 +435,233 @@
+        <p>
+          To <a>deproject</a> <a>depth map value</a> to point in depth camera
+          3D space, application or user agent must run following algorithm:
+        </p>
+        <p>
+          Let <var>dx</var> and <var>dy</var> be 2D coordinates, in pixels, of
+          a pixel in <a>depth map</a>.
+        </p><var></var>
+        <p>
+          Let <var>dz</var> be <a>depth map value</a> of the same pixel in the
+          <a>depth map</a>.
+        </p>
+        <p>
+          Let <var>fx</var> and <var>fy</var> be <a>depth map</a>'s
+          <a>horizontal focal length</a> and <a>vertical focal length</a>
+          respectively.
+        </p>
+        <p>
+          Let <var>cx</var> and <var>cy</var> be <a>depth map</a>'s
+          <a>principal point</a> 2D coordinates.
+        </p>
+        <p>
+          Let 3D coordinates (Xd, Yd, Zd) be the output of this step - a 3D
+          point in depth camera's 3D coordinate system.
+        </p>
+        <p>
+          `px = (dx - cx) / (fx)`
+        </p>
+        <p>
+          `py = (dy - cy) / (fy)`
+        </p>
+        <ul>
+          <li>If <a>depth map</a>'s <a>deprojection distortion coefficients</a>
+          are <a>not present</a> in <a href=
+          "#mediatracksettings-dictionary">MediaTrackSettings</a> dictionary,
+            <p>
+              3D coordinates (Xd, Yd, Zd) in depth camera space are calculated
+              as:
+            </p>
+            <p>
+              `Xd = dz * px`
+            </p>
+            <p>
+              `Yd = dz * px`
+            </p>
+            <p>
+              `Zd = dz`
+            </p>
+          </li>
+          <li>If <a>depth map</a>'s <a>deprojection distortion coefficients</a>
+          <a>k1</a>, <a>k2</a>, <a>k3</a>, <a>p1</a> and <a>p2</a> are
+          <a>present</a> in <a href=
+          "#mediatracksettings-dictionary">MediaTrackSettings</a> dictionary,
+          with a note that some of those could be zero,
+            <p>
+              3D coordinates (Xd, Yd, Zd) in depth camera space are calculated
+              as:
+            </p>
+            <p>
+              `r2 = px^2 + py^2`
+            </p>
+            <p>
+              `r = 1 + k1 * r2 + k2 * r2^2 + k3 * r2^3`
+            </p>
+            <p>
+              `Xd = dz * (px * r + 2 * p1 * px * py + p2 * (r2 + 2 * px^2))`
+            </p>
+            <p>
+              `Yd = dz * (py * r + 2 * p2 * px * py + p1 * (r2 + 2 * py^2))`
+            </p>
+            <p>
+              `Zd = dz`
+            </p>
+          </li>
+        </ul>
+        <p class="example">
+          In code, this condition would be like
+          <code>if(track.getSettings().deprojectionDistortionCoefficients)</code>
+        </p>
+      </section>
+      <section>
+        <h2>
+          <dfn>Transform</dfn> from depth to color 3D space
+        </h2>
+        <p>
+          The 3D point (Xd, Yd, Zd), that is a result of <a>project</a> depth
```

Comment:
`To <dfn>transform</dfn> xxxxxx, the user agent must run the <a>algorithm to transform yyyy</a>.`

`The <dfn>algorithm to transform yyyy</dfn> is as follows:`

(Remove dfn from the header.)

This allows us to decouple the "transform" concept from its algorithm, so we can link to them separately as needed elsewhere in the spec.